wiiu/vino/vino.ts

```diff
@@ -888,14 +975,14 @@ export default interface VinoAPI {
 	apd_isWiiuEnabled: () => boolean;
 
 	/**
-	 * Unknown
-	 * @returns Unknown
+	 * Gets time left for auto power down trigger
+	 * @returns Time left
 	 */
 	apd_getPeriod: () => number;
 
 	/**
-	 * Unknown
-	 * @returns Unknown
+	 * Gets last period where auto power down happened?
```

Comment:
If you're questioning what a description is, don't remove `unknown`. Only remove `unknown` if you actually know